### PR TITLE
PIA-1001: Regions search

### PR DESCRIPTION
**Pull request by kp-laura-sempere**
- Search functionality on the Regions list
- Navigate from the Regions section to the regions search list when the search button is selected

### CODE REVIEW

File: PIA VPN-tvOS/VPNConfigurationInstalling/Data/InstallVpnConfigurationProvider.swift [hunk 1]
```diff
@@ -18,17 +18,33 @@ class InstallVpnConfigurationProvider: InstallVPNConfigurationUseCaseType {
...
+    
     func callAsFunction() async throws {
         return try await withCheckedThrowingContinuation { continuation in
-            vpnProvider.install(force: true) { [self] error in
-                if error != nil {
-                    continuation.resume(throwing: InstallVPNConfigurationError.userCanceled)
-                    return
-                }
-                
+            if isSimulator {
```
Comment (kp-said-rehouni) [inline PIA VPN-tvOS/VPNConfigurationInstalling/Data/InstallVpnConfigurationProvider.swift:32]: nit: Maybe we could do this at the factory level. We can have a InstallVPNConfigurationProviderMock class which continues with the execution and at the factory level when we can check if it's simulator and inject the proper class.  
We could also use the strategy pattern an delegate the check to a strategy. In both cases we won't need to modify this class. What do you think @kp-laura-sempere?

**Review by kp-said-rehouni — COMMENTED** [review]
Great job @kp-laura-sempere! I just left a small comment that I think it will be interesting to look at.

File: PIA VPN-tvOS/VPNConfigurationInstalling/Data/InstallVpnConfigurationProvider.swift (hunk 1 above)
Comment (kp-laura-sempere) [reply PIA VPN-tvOS/VPNConfigurationInstalling/Data/InstallVpnConfigurationProvider.swift:32]: Yes, good idea, let me update the PR

**Review by kp-said-rehouni — APPROVED** [review]
LGTM! Great job @kp-laura-sempere!

**Review by kp-said-rehouni — APPROVED** [review]
LGTM! Good job @kp-laura-sempere!

**Pull request merged**